pom.xml

```diff
@@ -234,7 +209,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven.plugin-testing</groupId>
       <artifactId>maven-plugin-testing-harness</artifactId>
-      <version>2.1</version>
+      <version>3.3.0</version>
```

Comment:
3.1.0 max

Reply:
Done, where can I find the maximum version(s) not to exceed ?

Reply:
check https://maven.apache.org/plugin-testing/maven-plugin-testing-harness/dependency-management.html for maven-core dependency. The latest version depends on Maven 3.2.5